satpy/etc/areas.def

```diff
@@ -396,6 +396,15 @@ REGION: baws {
 	AREA_EXTENT:    (-475617, 5324430, 924383, 6724430)
 };
 
+REGION: baws_merc {
```

Comment:
Please remove your custom areas from this file. This file format is on it's way out anyway, and for custom area definition, we believe it's better to have them in a local file.

Reply:
Roger that!